### DISCUSSION TITLE
Siggraph Asia discussion on Dec 4th: AI in 5 or 10 years

### DISCUSSION THREAD

**Discussion by zhuo-yoyowz** [openvinotoolkit/openvino_notebooks#696] [Show and tell]
Let us know how you see AI in 5 to 10 years. And what role you would like to play in that journey.

Please leave your comments below. Thank you!

**Comment by siamiz88** [comment]
It will solve more complex non linear problem quickly, will find the solution even there will be sparse or abstract dataset, will be adapted to the domains that are known to indifferentiable. As a researcher, I eager to find the new approach along all three aspects that i mentioned.

**Reply by raymondlo84** [reply]
Oh! non-linear problems! yes
